### DISCUSSION TITLE
COVID-19 support

### DISCUSSION THREAD

**Discussion by orangemug** [maputnik/editor#877] [Announcements]
I lot of people have been impacted by COVID-19. Since this issue tracker has quite a few eyes on it, I thought it'd be another way to ask for help/support for those effected by the pandemic. If you have lost your job as a result of the pandemic or are working on a project related to the pandemic response, please feel free to add a comment below to ask for help. 

Please only add requests that are related to web mapping/geo in some way, because that's relevant to our audience. There will be better places to ask for help for other areas. I obviously don't know if this will have any real impact, but I thought if it helped spread your message to a few more people, or helped one person that's probably worth it.

To start the thread off I noticed that the Humanitarian OpenStreetMap Team has a post suggesting ways to help out https://www.hotosm.org/updates/mapping-in-the-times-of-covid-19-and-how-you-can-help/

**Comment by claustres** [comment]
Thanks for this initiative, we have taken some time to setup this project https://github.com/kalisio/covid-19 during the outbreak in order to aggregate and map data related to COVID-19, it's specific to **France** however. 

Good luck to all "mappers" in times of uncertainty !

**Comment by robert-claypool** [comment]
Is anyone looking for remote work? I need to hire someone to integrate Maputnik into a commercial SaaS product as [TomTom has done](https://github.com/maputnik/editor/issues/164#issuecomment-451662414). The hosting app is a fork of https://github.com/ant-design/ant-design-pro plus Apollo Client for API communications.

Please reach out to `RobertClaypool` on Twitter or email robert.claypool@fielda.com.

**Comment by nspringer-trimble** [comment]
Trimble MAPS has released a free progressive web app to alert truck drivers to closures of truck services and allow for them to report back any closures they find themselves. This was built using our own vector maps platform.
https://covid19.trimblemaps.com
https://blog.maps.trimble.com/map-intelligence-to-support-the-transportation-community
https://developer.trimblemaps.com
